bin/vtvalidate.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var exists = require('fs').existsSync;
+var fs = require('fs');
+var zlib = require('zlib');
+
+var usage = 'usage:';
+usage += '\n  node bin/vtvalidate.js <vector tile>';
+
+var input_tile = process.argv[2];
+if (!input_tile) {
+   console.log(usage);
+   process.exit(1);
+}
+
+if (!exists(input_tile)) {
+    console.log(input_tile + ' does not exist');
+    process.exit(1);
+}
+
+var validator = require('../');
```

Comment:
hey @springmeyer , curious how `../` is possible? I've been accessing the module itself via `../lib/index.js`. Is the module exposed magically to the root dir?

Reply:
I think it is possible due to the `main` property in https://github.com/mapbox/vtvalidate/blob/f0a99d721865015fb4be532c72f6c586df45ab37/package.json#L6. If you changed that `main` property to something else I presume this would fail. I don't have a strong preference for `../` but I've been in the habit of using it. I wonder if there is a best practice here...

Reply:
> due to the `main` property in
mmmm, great point!

After a quick google for best practices, can't seem to find anything specific atm. IMO, I like `../`. Seems relative paths can get sticky if things are copy-pasted or over-relied upon. Making sure `../` works via package.json seems solid 👍